sdk/react/adBar.js

```diff
@@ -88,11 +88,28 @@ var AdBar = React.createClass({
     return textString;
   },
 
+  getSkipTimerLabel: function() {
+    var skipTimerLabel = "";
```

Comment:
can we move this to utils? conversion between time number and string should be reusable.